### ISSUE TITLE
"Highlight" point on scatter plot

### ISSUE THREAD

**Issue by Benny121221** [swharden/ScottPlot#414]
**What can we do to make ScottPlot better?**
![image](https://user-images.githubusercontent.com/8635304/82615033-9111b080-9b86-11ea-8312-6f41923eca7a.png)
In this image I overlayed another scatterplot with a different colour and a larger marker size. It would perhaps be nicer to do this without adding another plottable?

Does your proposed feature allow users to do new things in source code? If so, show us how you'd like to be able to use the new feature:

```cs
var my_scatter.higlight(x,y);
```

**Comment by swharden** [comment]
Interesting idea! I like it. Maybe it could also be used like...

```cs
myScatter.HighlightClear();
myScatter.HighlightPoint(pointIndex);
myScatter.HighlightPointNearest(x, y);
```

I'll think about this! The best way to experiment with it may to be to copy the `Scatter` plottable and call it something like `ScatterHighlight`. There would be many several extra options for `highlightedPointSize`, `highlightedPointShape`, and `highlightedPointColor` and I'd rather not drag all those into `Scatter` (which I'd like to keep simple and light).

If you want to give it a go you may find https://github.com/swharden/ScottPlot/blob/master/CONTRIBUTING.md#create-a-new-plottable-object helpful while I work on other things (#412)

**Comment by bukkideme** [comment]
Would be very useful! I use such "markers" as secondary scatterplot with different color now, but this is a better way.

**Comment by swharden** [comment]
Merged! #415 Thanks @Benny121221 and @StendProg
